ERCS/erc-7625.md

```diff
@@ -0,0 +1,577 @@
+---
+eip: 7625
+title: Smart Contract Id's and Metadata Extension
+description: Proposes an interface for contractId's and metadata at the smart contract level.
```

Comment:
Your description doesn't add any new information that wasn't in the title, plus all ERCs are proposing something, and most are describing interfaces. In short, you have a lot of fluff in your description.